observer/include/observer/parameters/RobotParameterDatabase.h

```diff
@@ -0,0 +1,31 @@
+//
+// Created by rolf on 21-10-20.
+//
+
+#ifndef RTT_ROBOTPARAMETERDATABASE_H
+#define RTT_ROBOTPARAMETERDATABASE_H
+#include <roboteam_proto/messages_robocup_ssl_referee.pb.h>
+#include <data/RobotParameters.h>
+struct TwoTeamRobotParameters{
+    bool blueChanged = false;
+    bool yellowChanged = false;
+    RobotParameters blueParameters;
+    RobotParameters yellowParameters;
+    [[nodiscard]] proto::TeamParameters yellowTeamProto() const;
+    [[nodiscard]] proto::TeamParameters blueTeamProto() const;
+};
+class RobotParameterDatabase {
+public:
+    TwoTeamRobotParameters update(const proto::SSL_Referee& refMessage);
+    [[nodiscard]] TwoTeamRobotParameters getParams() const;
+
+    static RobotParameters getTeamParameters(const std::string& teamName);
+private:
+    std::string blueName;
+    RobotParameters blueParameters;
+    std::string yellowName;
+    RobotParameters yellowParameters;
+};
```

Comment:
can we run clang-format over the entire codebase when we finish the refactor?

Reply:
Yes, agreed. My IDE somehow doesn't get the settings half of the time, unfortunately.